build/azuredevops-taskctl.yml

```diff
@@ -24,6 +24,10 @@ stages:
     jobs:
       - job: Build
         steps:
+          # Do not perform a shallow checkout of the repo
+          - checkout: self
+            fetchDepth: 0
```

Comment:
Do you want this any more if you've disabled the build stamp?

Reply:
Good point - will remove it now